src/components/layout/OrgHeader.tsx

```diff
@@ -9,11 +9,10 @@ import {
     View,
 } from '@adobe/react-spectrum';
 
-import apiUrl from '../../utils/apiUrl';
 import { ZetkinOrganization } from '../../interfaces/ZetkinOrganization';
 
 interface OrgHeaderProps {
-    org: ZetkinOrganization;
+    org?: ZetkinOrganization;
```

Comment:
Why allow `org` to be undefined here? I feel like that should never happen, because if we are using `OrgHeader` that's because we are rendering a page where the organization should always be available.

Reply:
This might have had something to do with the last issue we fixed? Works without `?`!